ebu_tt_live/scripts/ebu_ebuttd_encoder.py

```diff
@@ -34,13 +38,28 @@
                     help='Works only with -m, if set the script will wait for new lines to be added to the file once the last line is reached. Exactly like tail -f does.',
                     action="store_true", default=False
                     )
+parser.add_argument('--implicit-ns', help='Some tools hardcode tt so they can\'t understand tt:tt so global namespace disappears',
+                    action='store_true', dest='implicit_ns', default=False)
 parser.add_argument('-z', '--clock-at-media-time-zero', dest='media_time_zero',
                     help='This sets the offset value that is used to turn clock time into media time.',
                     default='current', metavar='HH:MM:SS.mmm')
+parser.add_argument('-zo', '--clock-at-media-time-zero-offset', dest='media_time_zero_offset',
+                    help='This sets the offset value that is used to turn clock time into media time.',
+                    default='current', metavar='HH:MM:SS.mmm')
+parser.add_argument('-ss', '--segmentation-starts', dest='segmentation_starts',
+                    help='This helps with local machine clock timing adjustment',
```

Comment:
I agree. This does not make any sense. I don't know what I was thinking. This is a time mapping that I whipped up very quickly when we had 6 seconds lag behind or ahead of the ericsson computer